scripts/shell/change_app.toml.sh

```
@@ -0,0 +1,69 @@
+# 1. Enables telemetry with hostname and service labels
+# 2. Sets Prometheus metrics retention to 100 blocks
+# 3. Activates API, Swagger UI, and CORS
+#
+# Targets:
+# - Validators 1-6
+# - RPC nodes 1-2
+# - Bootnodes 1-2
+#############################################################################
+
+# Navigate to primary validator's config directory
+cd /data/genesis-validator-1/.treasurenetd/config/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)